examples/calendar/backend/privacy_cir/verification_key.json

```diff
@@ -0,0 +1,140 @@
+{
+	"protocol": "groth16",
+	"curve": "bn128",
+	"nPublic": 12,
```

Comment:
in the `setup.sh` you set `nPublic` to 14 and here you use 12?

Reply:
This was generated by circom, not done manually.